README.rst

```diff
@@ -1139,6 +1139,10 @@ Choose from the list of available rules:
 
   There MUST be no trailing spaces inside comment or PHPDoc.
 
+* **no_unneeded_alias**
```

Comment:
I would name it `no_unneeded_import_alias` to prevent any ambiguity with other kinds of alias.